src/build/vita-public.h

```diff
@@ -0,0 +1,9 @@
+#ifndef VITA_PUBLIC
```

Comment:
You may name such as `VITA_TOOLCHAIN_PUBLIC`, but I think it's obvious enough if you see the header.